momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -829,16 +828,14 @@ public CompletableFuture<CacheSortedSetPutElementsResponse> sortedSetPutElements
    * @param endRank - The rank of the last element to fetch. This rank is exclusive, ie the element
    *     at this rank will not be fetched. Defaults to null, which fetches up until and including
    *     the last element.
-   * @param order - The order to fetch the elements in. Defaults to ascending.
    * @return Future containing the result of the fetch operation.
    */
   public CompletableFuture<CacheSortedSetFetchResponse> sortedSetFetchByRank(
       String cacheName,
       String sortedSetName,
       @Nullable Integer startRank,
-      @Nullable Integer endRank,
-      @Nullable SortOrder order) {
-    return scsDataClient.sortedSetFetchByRank(cacheName, sortedSetName, startRank, endRank, order);
+      @Nullable Integer endRank) {
```

Comment:
removing the SortOrder seems wrong to me.  Don't we have to tell the server the sort order so that the indices we are asking for will be correct?  Or is it applying the indices first and then applying the sort order afterward (in which case what you are doing here is probably okay)?

If you know the answer to this can you please update the notes here in the spec:

https://www.notion.so/momentohq/SDK-API-Methods-Spec-cf12e4db2535405ca2acc3608814b083#238e5fc189a5498392e9dc2d0fd45e8b

and maybe also check the public dev docs to make sure they are clear about this?

Reply:
I hadn't considered the interaction between sort order and the indices. If that's the case then I think we need to use a list instead of a NavigableSet since the set will have the same default sort order regardless of what order the user wants.